hie-base/Haskell/Ide/Engine/PluginTypes.hs

```diff
@@ -18,8 +18,14 @@ module Haskell.Ide.Engine.PluginTypes
   , ParamValP(..)
   , ParamMap
   , pattern ParamTextP
+  , pattern ParamIntP
+  , pattern ParamBoolP
   , pattern ParamFileP
```

Comment:
We should possibly remove ParamFileP and ParamPosP, as superseded by the LSP types